app/scripts/controllers/newOverview.js

```diff
@@ -1221,15 +1225,22 @@ function OverviewController($scope,
         resource: 'instances'
       }, context, function(serviceInstances) {
         state.serviceInstances = serviceInstances.by('metadata.name');
+        updateFilter();
       }, {poll: limitWatches, pollInterval: DEFAULT_POLL_INTERVAL}));
     }
 
     if(Constants.ENABLE_TECH_PREVIEW_FEATURE.service_catalog_landing_page) {
+
+      DataService.list("secrets", context, null, { errorNotification: false }).then(function(secretData) {
```

Comment:
need to think about how we are loading these, any time you create a new binding that is going to create new secrets.  This probably either needs to be a limited watch, or we need to refresh the list any time we see an update to bindings (debounced so it doesn't spam the server with requests if there are a lot binding updates at once).  I'm leaning towards the latter.  @spadgett thoughts?